tests/jvm/src/test/scala/scala/meta/tests/semanticdb/ExpectSuite.scala

```diff
@@ -107,7 +108,18 @@ trait ExpectHelpers extends FunSuiteLike {
       if PathIO.extension(file.toNIO) == "semanticdb"
       doc <- s.TextDocuments.parseFrom(file.readAllBytes).documents
       sym <- doc.symbols
-    } yield sym.symbol -> sym
+      normalized = sym.signature match {
+        case c: s.ClassSignature =>
+          sym.copy(
+            signature = c.copy(
+              declarations = Some(
+                c.declarations.get.copy(symlinks = c.declarations.get.symlinks.sorted)
+              )
+            )
+          )
+        case _ => sym
+      }
```

Comment:
Why is this necessary?

Reply:
I opened https://github.com/scalameta/scalameta/issues/1642 to track this change so that we can revert it later. At this moment the noisy diffs from declaration ordering made it difficult to spot actual bugs.

Reply:
Sounds good.